src/main/java/com/swordglowsblue/artifice/api/builder/data/dimension/BiomeSourceBuilder.java

```diff
@@ -131,7 +137,13 @@ public BiomeParametersBuilder altitude(float altitude) {
              * @return
              */
             public BiomeParametersBuilder weirdness(float weirdness) {
-                this.root.addProperty("weirdness", weirdness);
+                try {
+                    if (weirdness > 2.0F) throw new Throwable("weirdness can't be higher than 2.0F! Found " + weirdness);
```

Comment:
What is this? this will just get catched and print an exception stack trace immediately. Use an `IllegalArgumentException`